### PR TITLE
tuya-panel-kit: add navigationContainerProps param to createNavigator

### PR DESCRIPTION
**Pull request by youngjunning**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/tuya-panel-kit/index.d.ts [hunk 1]
```diff
@@ -11323,7 +11323,7 @@ export interface NavigationComponentClass<P = {}, S = {}> {
     new (props: P, context?: any): NavigatorLayout<P, S>;
 }
 // tslint:disable-next-line no-unnecessary-generics
-export function createNavigator<P = {}, S = {}>(createNavigatorParam: NavigationParam): NavigationComponentClass<P, S>;
+export function createNavigator<P = {}, S = {}>(createNavigatorParam: NavigationParam, navigationContainerProps: Omit<NavigationContainerProps, 'children'>): any;
```
Comment (amcasey) [inline types/tuya-panel-kit/index.d.ts:11326]: I'm surprised to see a new required parameter without a version bump.  Maybe this was intended to be optional?  And what happened to the return type?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/tuya-panel-kit/index.d.ts (hunk 1 above)
Comment (youngjuning) [reply types/tuya-panel-kit/index.d.ts]: Thanks for review, I have bump the version and set new parameter as optional.

**Review by amcasey — APPROVED** [review]
Thanks!

**Review by ShinyLeee — APPROVED** [review]

**Comment by youngjuning** [comment]
Ready to merge

**Pull request merged**